doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,6 @@
         "#1": "Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For a sqlite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Set \"serverVersion\" to your server version to avoid edge-case exceptions and extra database calls",
-        "DATABASE_URL": "mysql://root@127.0.0.1:3306/symfony?charset=utf8mb4&serverVersion=5.7"
+        "DATABASE_URL": "mysql://app:app@127.0.0.1:3306/app?charset=utf8mb4&serverVersion=5.7"
```

Comment:
@teohhanhui having a variable called `DATABASE_URL` and containing a DSN is a fairly common convention popularized by Rails.

It's used by:

* Rails (of course): http://edgeguides.rubyonrails.org/configuring.html#configuring-a-database
* Django Environ: https://django-environ.readthedocs.io/en/latest/#how-to-use
* Heroku: https://devcenter.heroku.com/articles/heroku-postgresql#create-a-new-database
* Cloud Foundry: https://docs.cloudfoundry.org/buildpacks/ruby/ruby-service-bindings.html#configuring-non-rails-applications

I'm to keep this as is.